core/block_svg.js

```diff
@@ -794,6 +794,23 @@ Blockly.BlockSvg.prototype.setShadow = function(shadow) {
   this.updateColour();
 };
 
+/**
+ * Set whether this block is an insertion marker block or not.
+ * @param {boolean} insertionMarker True if an insertion marker.
+ * @package
+ */
+Blockly.BlockSvg.prototype.setInsertionMarker = function(insertionMarker) {
+  if (this.isInsertionMarker_ == insertionMarker) {
+    return;  // No change.
+  }
+  this.isInsertionMarker_ = insertionMarker;
+  if (this.isInsertionMarker_) {
```

Comment:
The current implementation means it can't be undone, but the API implies you can set it false again.

Add a comment on the API that once its been set to true unsetting it is not supported.

Reply:
Done.